### PR TITLE
Realisation of task3, homework 2

### PR DESCRIPTION
**Pull request by EvgenyBelichkov**
Can your check my code?
The task is: "Write a function that takes K lists as arguments and returns all possible lists of K items where the first element is from the first list, the second is from the second and so one. You may assume that that every list contain at least one element"
Thank you in advance.

### CODE REVIEW

File: homework2/task03/tests/test_of_possible_combinations.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from homework2.task03 import all_possible_lists
+
+
+@pytest.fixture
+def fixture():
```
Comment (i-n-samartsev) [inline homework2/task03/tests/test_of_possible_combinations.py]: ну я же говорил исправить подход к фикстурам. Семен Семеныч..

**Comment by i-n-samartsev** [comment]
Исправь тесты

**Review by i-n-samartsev — CHANGES_REQUESTED** [review]
надо исправить конфликты

**Comment by EvgenyBelichkov** [comment]
Code will be recommited

**Pull request closed without merging**